queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name  LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] Kindly fix this typo error in the query `List the name of all animals that are neutered and have less than 3 escape attempts` for the `escap_attempts` as it's called `escape_attempt` in your schema.sql file.